### PR TITLE
[LF-1451] crop variety page filter

### PR DESCRIPTION
**Pull request by beetai**

### CODE REVIEW

File: packages/webapp/src/containers/Filter/CropVariety/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  setCropVarietyFilterDefault,
+} from '../../filterSlice';
+import { suppliersSelector } from '../../cropVarietySlice';
+
+const statuses = [ACTIVE, ABANDONED, PLANNED, COMPLETE, NEEDS_PLAN];
+
+const CropVarietyFilterPage = ({ cropId, onGoBack }) => {
+  const { t } = useTranslation(['translation', 'filter']);
+  const cropEnabledLocations = useSelector(cropLocationsSelector);
+  // const cropCatalogueFilter = useSelector(cropCatalogueFilterSelector);
```
Comment (kikemarquez8) [inline packages/webapp/src/containers/Filter/CropVariety/index.js:29]: comment

File: packages/webapp/src/containers/CropCatalogue/useFilterVarieties.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { LOCATION, STATUS, SUPPLIERS } from '../Filter/constants';
+import { cropVarietyFilterSelector } from '../filterSlice';
+
+export default function useFilterVarieties(varieties, cropId, status) {
+  const cropVarietyFilter = useSelector(cropVarietyFilterSelector(cropId));
+  // if (!cropVarietyFilter) return varieties;
```
Comment (kikemarquez8) [inline packages/webapp/src/containers/CropCatalogue/useFilterVarieties.js:8]: comment

File: packages/webapp/src/containers/CropCatalogue/useFilterVarieties.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+    if (included.size === 0) return filteredBySupplier;
+    return varieties.filter((variety) => included.has(variety.location_id));
+  }, [cropVarietyFilter?.[LOCATION], filteredBySupplier]);
+
+  const filteredByStatus = useMemo(() => {
+    if (!cropVarietyFilter) return filteredByLocation;
+    const statusFilter = cropVarietyFilter[STATUS];
+    const activeFilterSuppliers = new Set(
+      Object.keys(statusFilter).filter((status) => statusFilter[status].active),
```
Comment (kikemarquez8) [inline packages/webapp/src/containers/CropCatalogue/useFilterVarieties.js:35]: Why didn't you do this in the other 2 filters?

Comment (beetai) [reply packages/webapp/src/containers/CropCatalogue/useFilterVarieties.js:35]: oh oops let me do that

**Review by kikemarquez8 — APPROVED** [review]

**Pull request merged**